config/cluster-baremetal-operator/kustomization.yaml

```diff
@@ -1,7 +1,13 @@
 commonAnnotations:
+  capability.openshift.io/name: baremetal
   include.release.openshift.io/self-managed-high-availability: "true"
   include.release.openshift.io/single-node-developer: "true"
-  capability.openshift.io/name: baremetal
 
 resources:
 - cluster-baremetal-operator.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I wonder why this gets added. Will the CI pass if you drop the changes to this file?

Reply:
Ah yes, changes to this file must be because I ran some make command locally. I think they should be dropped, will do and retest.